src/main/java/Lotto/WinningPrice.java

```diff
@@ -0,0 +1,31 @@
+package Lotto;
+
+import java.util.Arrays;
+
+
+public enum WinningPrice {
+
+
+    THREE_WINNINGS(3, 5000),
+    FOUR_WINNINGS(4, 50000),
+    FIVE_WINNINGS(5, 1500000),
+    SIX_WINNINGS(6, 2000000000);
```

Comment:
```suggestion
    SIX_WINNINGS(6, 2_000_000_000);
```
다음과 같은 표현식으로 가독성을 좀 더 확보할 수 있습니다!

Reply:
감사합니다 수정하겠습니다 👍